docs/docker-backend-nemesis.md

```diff
@@ -0,0 +1,91 @@
+# Individual Nemesis cases status on Docker backend
+| Nemesis | Status | Comment
+| :------ | :----- | :------
+| AbortRepairMonkey | Pass |
+| AddDropColumnMonkey | Pass |
+| AddRemoveDcNemesis  | Skipped | Requires multi-dc env
+| AddRemoveMvNemesis | Pass |
+| AddRemoveRackNemesis | Pass |
+| BlockNetworkMonkey | Pass |
+| BootstrapStreamingErrorNemesis | Pass |
+| CDCStressorMonkey | Pass |
+| ClusterRollingRestartRandomOrder | Pass |
+| ClusterRollingRestart | Pass |
+| CorruptThenRebuildMonkey | Pass |
+| CorruptThenRepairMonkey | Pass |
+| CorruptThenScrubMonkey | Pass |
+| CreateIndexNemesis | Pass |
+| DecommissionMonkey | Pass |
+| DecommissionSeedNode | Fail | [scylla-cluster-tests/issues/7328](https://github.com/scylladb/scylla-cluster-tests/issues/7328)
+| DecommissionStreamingErrMonkey | Pass |
+| DeleteByPartitionsMonkey | Pass |
+| DeleteByRowsRangeMonkey | Pass |
+| DeleteOverlappingRowRangesMonkey | Pass |
+| DisableBinaryGossipExecuteMajorCompaction | Pass |
+| DrainerMonkey | Pass |
+| DrainKubernetesNodeThenDecommissionAndAddScyllaNode | Skipped | Not a k8s backend
+| DrainKubernetesNodeThenReplaceScyllaNode | Skipped | Not a k8s backend
+| EnableDisableTableEncryptionAwsKmsProviderWithoutRotationMonkey | Skipped | Not an AWS backend
+| EnableDisableTableEncryptionAwsKmsProviderWithRotationMonkey    | Skipped | Not an AWS backend
+| EndOfQuotaNemesis | Skipped | Should be skipped for docker backend
+| EnospcAllNodesMonkey | Pass |
+| EnospcMonkey | Pass |
+| GrowShrinkClusterNemesis | Pass |
+| HardRebootNodeMonkey | Skipped | Backend doesn't support hard reboot
```

Comment:
Should not be difficult to implement it with:
```
docker kill <container_id>
docker start <container_id>
<wait for container startup>
```
Could be planned as separate task

Reply:
Created a task to add support for hard reboot in Docker backend - https://github.com/scylladb/qa-tasks/issues/1661